README.md

```diff
@@ -62,6 +62,39 @@ You can start training pipelines from your local computer by creating an environ
   - `python -m mlops.common.register_data_asset --data_config_path config/data_config.json`
 - Run the training pipeline under test using the module notation (for example, `python -m mlops.nyc_taxi.start_local_pipeline --build_environment pr --wait_for_completion True`)
 
+## Running Debug Tasks in VS Code
+
+You can use Visual Studio Code to run and debug specific tasks related to the MLOps pipelines. The following configurations are set up in the [launch.json](.vscode/launch.json) file, allowing you to execute various scripts with ease.
+
+### Available Debug Tasks
```

Comment:
I would remove the `--build_environment` "pr" value, and instead instruct the user to provide a build_environment to use (have the task prompt the user for it)